rqt_py_common/src/rqt_py_common/rqt_roscomm_util.py

```diff
@@ -199,12 +213,14 @@ def mfilter(f):
 
     @staticmethod
     def is_roscore_running():
```

Comment:
Should functions like these be deprecated or just removed?

Reply:
I've been leaving the ones that I think may be ported in the future. The rest can be deleted